adapters/unicorn/unicorn.go

```diff
@@ -64,6 +64,9 @@ func (a *adapter) MakeRequests(request *openrtb2.BidRequest, requestInfo *adapte
 		}
 	}
 
+	// Support Cur "JPY" only
+	request.Cur = []string{"JPY"}
```

Comment:
Do you think it would be a good idea to return an error if a non JPY currency is specified? .. or would you consider trying to use the built-in currency converter?

Reply:
we want to support only the currency JPY, we chose the former (i.e. return an error if a non JPY currency is specified).

But when no currency is specified, we specify currency JPY as default.

https://github.com/prebid/prebid-server/pull/2265/commits/771617d0a57024db1e94f1f72ff5564503d34212